### PR TITLE
Fix broken links in Kotlin DSL primer

### PR DESCRIPTION
**Pull request by eskatos**
Fixes https://github.com/gradle/gradle/issues/15874

### Contributor Checklist
- [ ] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [ ] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [ ] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [ ] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [ ] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [ ] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [ ] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [ ] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [ ] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation
- [ ] Recognize contributor in release notes

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test this

**Review by Vampire — CHANGES_REQUESTED** [review]
What about the maven-plugin sample?
grafik

**Comment by eskatos** [comment]
Good catch @Vampire, thanks!
Fixed, could you please take another look?

**Comment by eskatos** [comment]
@bot-gradle test this

File: subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc [hunk 1]
```diff
@@ -939,7 +936,7 @@ include::sample[dir="snippets/kotlinDsl/interoperability-groovy-builder/kotlin",
 <4> Configure the `blockName` property, maps to a `Closure` taking method invocation
 <5> Invoke `another` method taking named arguments, maps to a Groovy named arguments `Map<String, ?>` taking method invocation
 
-The link:{kotlin-dsl-samples}maven-plugin[maven-plugin] sample demonstrates the use of the `withGroovyBuilder()` utility extensions for configuring the `uploadArchives` task to <<maven_plugin#sec:deploying_to_a_maven_repository, deploy to a Maven repository>> with a custom POM using Gradle's core <<maven_plugin#, Maven Plugin>>.
+The <<maven_plugin#sec:deploying_to_a_maven_repository, deploying to a Maven repository>> using Gradle's core <<maven_plugin#, Maven Plugin>> demonstrates the use of the `withGroovyBuilder()` utility extensions for configuring the `uploadArchives` task with a custom POM.
```
Comment (Vampire) [inline subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc:939]: While you are at it, is there maybe a better example than the deprecated `maven` plugin? :-)

**Pull request merged**